docs/api-reference.md

```diff
@@ -162,6 +162,14 @@ The request payload is in JSON, and may contain the `options` field. The `option
 * `inverseEnabled` (boolean): Enables [the inverted view]({{< ref "data-model.md#inverse" >}}) for this frame if `true`.
 * `cacheType` (string): [ranked]({{< ref "data-model.md#ranked" >}}) or [LRU]({{< ref "data-model.md#lru" >}}) caching on this frame. Default is `lru`.
 * `cacheSize` (int): Number of rows to keep in the cache. Default 50,000.
+* `rangeEnabled` (boolean): Enables range encoded numbers in this frame.
+* `fields` (array): List of range encoded fields.
+
+Each individual `field` contains the following:
+* `name` (string): Field name.
+* `type` (string): Field type, currently only "int" is supported.
+* `min` (int): Minimum of the value range stored in this field.
+* `max` (int): Maximum of the value range stored in this field.
```

Comment:
What is the range of allowed values of max and min? In particular, are negative values allowed?

Reply:
We might point out that a field will use `ceil(log2(max-min))` bits for the value, so the user could maximize the allowed values by using a power of two range.

Reply:
Which of these can be modified, if any?

Reply:
Yes, negative values are allowed. Min and max can be anything between `MinInt64` and `MaxInt64`. These values currently cannot be modified. Changing `min` would require re-encoding everything. Increasing `max` could be possible. `name` could be changed too but it requires renaming all the views.